k8s-training/Readme.md

```diff
@@ -70,15 +83,29 @@ Build image
 packer build .
 ```
 
-### Build training image
+Use the resulting image as base image for the dockerfile
+
+### GCP-GKE
+
+For Fast Socket gNIC enabled base image use a Deep Learining VM Image as base image.
+
+:information_source: To view list of container images available `gcloud container images list \
+--repository="gcr.io/deeplearning-platform-release"
+`
+
+## Build training image
 
 ```bash
-docker built -t k8s-training/pytorch:v1.12.0 .
+# For EKS with EFA support
+docker built --build-arg BASE_IAMGE=<Image-Built-From-Above-Packer-Script> -t k8s-training/pytorch:v1.12.0 .
+
```

Comment:
Once the `packer build .` command is successful, docker image is getting created without a name and version.

For example

```
(base) ubuntu@ip-172-31-22-204:~$ docker images
REPOSITORY            TAG                               IMAGE ID       CREATED        SIZE
<none>                <none>                            c2bff6e05f6b   4 hours ago    20.9GB
```

Before moving to the next step(build training image). This image needs to be tagged appropriately

Can you please add instructions for the same ?